CHANGELOG.md

```diff
@@ -47,15 +47,15 @@
  - asPercent may be used as an aggregator (#2594, @replay)
  - Make nodes in group by nodes optional (#2597, @replay)
  - privatize linearRegressionAnalysis so it will not show in documentation (#2602, @piotr1212)
- - Revert symlink fix (#2604, @piotrt1212)
+ - Revert symlink fix (#2604, @piotr1212)
```

Comment:
I'm pretty sure about each of these user changes.

That said, some projects don't like changing their changelogs. I'm happy to drop any changes as directed.